src/ModelContextProtocol/Protocol/Messages/RequestMethods.cs

```diff
@@ -0,0 +1,82 @@
+﻿namespace ModelContextProtocol.Protocol.Messages;
+
+/// <summary>
+/// Provides names for request methods used in the Model Context Protocol (MCP).
+/// </summary>
+public static class RequestMethods
+{
+    /// <summary>
+    /// Sent from the client to request a list of tools the server has.
+    /// </summary>
+    public const string ToolsList = "tools/list";
```

Comment:
Another reason why I like this is because it offers a condensed overview of what is already implemented.